relay-server/src/services/store.rs

```diff
@@ -896,13 +902,47 @@ impl StoreService {
                     false
                 }
             });
-        }
 
-        span.duration_ms = ((span.end_timestamp - span.start_timestamp) * 1e3) as u32;
-        span.event_id = event_id;
-        span.project_id = scoping.project_id.value();
-        span.retention_days = retention_days;
-        span.start_timestamp_ms = (span.start_timestamp * 1e3) as u64;
+            let group = span
+                .sentry_tags
+                .as_ref()
+                .and_then(|sentry_tags| sentry_tags.get("group"))
+                .unwrap_or(&"");
+
+            for (mri, summaries) in metrics_summary {
+                let Some(summaries) = summaries else {
+                    continue;
+                };
+                for summary in summaries {
+                    let Some(summary) = summary else {
+                        continue;
+                    };
+                    // Ignore immedate errors on produce.
+                    let _ = self.produce(
+                        KafkaTopic::MetricsSummaries,
+                        scoping.organization_id,
+                        KafkaMessage::MetricsSummary(MetricsSummaryKafkaMessage {
+                            count: summary.count.unwrap_or_default(),
```

Comment:
I assume that the fields of `MetricsSummary` are `Optional` for a reason. IMO we should do one of two things:

1. Make the fields of `MetricsSummary` non-optional.
2. If it actually makes sense that these fields are empty, make the consumer accept missing / `null` values.

Reply:
I think I forgot where I was going with this. The consumer supports missing values but not `null` values and I think I wanted to turn that into non-optional values.